src/stories/list.stories.js

```diff
@@ -50,11 +46,33 @@ export const ListA = () => {
 
     return (
         <StoriesContainer>
+            <p>Profile takes key/value - value as string or number only These are dumb componets</p>
```

Comment:
This story is about `List` and `ListItem`, not Profile.
These components are simple stateless components.
`List` accept `ListItem`\`s and `ListItem` accept a pre-defiend `type` and primitive `value` of a string or number type